src/app/components/common/node-editor/node-editor.component.html

```diff
@@ -68,29 +82,77 @@
         </div>
       </div>
     </ng-container>
-    <ng-container *ngIf="showLogs">
+    <ng-container *ngIf="showLogs && from !== 'notifications'">
       <app-system-log [sourceName]="serviceName"></app-system-log>
     </ng-container>
+
+    <!-- Notification Log in Quickview -->
+    <ng-container *ngIf="showLogs && from === 'notifications'">
+      <app-notification-log [sourceName]="serviceName"></app-notification-log>
+    </ng-container>
+
+    <!-- Notification Instance Configuration in Quickview -->
```

Comment:
Better to move these changes in a separate component

Reply:
We can handle it in the next patch PR.